tests/PHPUnit/OneOfConstraint.php

```diff
@@ -33,6 +33,8 @@ public function __construct(array $array)
     protected function matches($other): bool
     {
         if (is_array($other)) {
+            $other = is_array($other[0]) ? array_merge(...$other) : $other;
```

Comment:
I don't know what this does. Removing for now.

Reply:
Basically, relay is sending the data in the form of an array inside an array, this check if the first element of the array is array then it flatten the array into one dimensional array

Reply:
I think the better solution was this: 5d729787f0f153feac03c503f91be67c831de72d